bin/node/cli/src/browser.rs

```diff
@@ -15,45 +15,27 @@
 // along with Substrate.  If not, see <http://www.gnu.org/licenses/>.
 
 use crate::ChainSpec;
-use futures01::{prelude::*, sync::oneshot, sync::mpsc};
-use libp2p::wasm_ext;
-use log::{debug, info};
-use std::sync::Arc;
-use sc_service::{AbstractService, RpcSession, Roles as ServiceRoles, Configuration, config::DatabaseConfig};
+use log::info;
 use wasm_bindgen::prelude::*;
+use sc_service::Configuration;
+use browser_utils::{Transport, browser_configuration, set_hooks, Client};
 
 /// Starts the client.
-///
-/// You must pass a libp2p transport that supports .
 #[wasm_bindgen]
-pub fn start_client(wasm_ext: wasm_ext::ffi::Transport) -> Result<Client, JsValue> {
+pub async fn start_client(wasm_ext: Transport) -> Result<Client, JsValue> {
 	start_inner(wasm_ext)
+		.await
 		.map_err(|err| JsValue::from_str(&err.to_string()))
 }
 
-fn start_inner(wasm_ext: wasm_ext::ffi::Transport) -> Result<Client, Box<dyn std::error::Error>> {
-	console_error_panic_hook::set_once();
-	console_log::init_with_level(log::Level::Info);
+async fn start_inner(wasm_ext: Transport) -> Result<Client, Box<dyn std::error::Error>> {
+	set_hooks(log::Level::Debug);
```

Comment:
Are you intentionally setting a different log level then we had before?

Reply:
No, that was an experimental change and I forgot to change it back